jax/_src/lru_cache.py

```diff
@@ -153,26 +171,36 @@ def _evict_if_needed(self, *, additional_size: int = 0) -> None:
     if not self.eviction_enabled:
       return
 
-    # a priority queue, each element is a tuple `(file_mtime, file, file_size)`
-    h: list[tuple[int, pathlib.Path, int]] = []
+    # a priority queue, each element is a tuple `(file_atime, key, file_size)`
```

Comment:
You can use `typing.NamedTuple` to describe a structure of this tuple instead of a comment.

This comment is optional.

Reply:
I think `NamedTuple` would be verbose here